irohad/ordering/impl/ordering_gate_impl.cpp

```diff
@@ -53,15 +47,16 @@ namespace iroha {
       }
       log_->info("transactions in proposal: {}", transactions.size());
 
-      model::Proposal proposal(transactions);
-      proposal.height = request->height();
-      handleProposal(std::move(proposal));
+      auto proposal = transport_->getProposal(request);
 
+      proposal->height = request->height();
+      transport_->subscriber_->handleProposal(proposal);
+      //handleProposal(proposal);
```

Comment:
Why does the received proposal go to transport again?

Reply:
first it gets a proposal from request, then it handles it